lms/urls.py

```diff
@@ -429,6 +429,12 @@
             include('edxnotes.urls'), name="edxnotes_endpoints"),
     )
 
+    if settings.FEATURES["ENABLE_TEAMS"]:
```

Comment:
That seems reasonable. We need to design how we are going to support configuration-based feature flags so that we do it in a consistent way for all features going forward, so this should be its own story.